### ISSUE TITLE
Could not find a version that satisfies the requirement numpy~=1.22.0 

### ISSUE THREAD

**Issue by robmarkcole** [canturan10/satellighte#14]
Hi
git cloned and attempt `pip install .` in a google colab instance results in the error:

```
Could not find a version that satisfies the requirement numpy~=1.22.0 
```

Likely versions need updating. 

As an experiment I removed all the version guides and then get `ERROR: Package 'satellighte' requires a different Python: 3.7.14 not in '>=3.8'` which I guess needs google to update colab, however updating in `setup.py` to `   python_requires=">=3.7",` I can install successfully and run `python3 training/eurosat_training.py `

**Comment by canturan10** [comment]
Hello There 👋



Latest numpy does not support python 3.7 anymore. You’ll notice that the files are built for Python 3.8, 3.9, and 3.10. 
https://pypi.org/project/numpy/1.22.0/#files

And the reason why I upgraded to the newest version:
![image](https://user-images.githubusercontent.com/34894012/191895497-b526a07d-4f5e-432f-b930-8bd64cb047c9.png)

You have already discovered the problem while I am writing the answer :) If we do as you say, yes it will work (previous versions of the library were like this), but I had to update due to security weakness. I hope I could help.

Thank you.
